### PR TITLE
chore: We love the FieldSelector component

### PR DESCRIPTION
**Pull request by RyanSkraba**
and don't want to delete it.

**What is the current behavior?** (You can also link to an open issue here)

FieldSelector component deleted.

**What is the new behavior?**

FieldSelector component not deleted.

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/1776/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/1776/)

File: components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/ProcessingFamilyDefinition.java [hunk 1]
```diff
@@ -45,7 +46,9 @@ public ProcessingFamilyDefinition() {
         super(NAME,
                 // Components
                 new FilterRowDefinition(), new PythonRowDefinition(), new ReplicateDefinition(),
-                new TypeConverterDefinition(), new WindowDefinition(), new NormalizeDefinition(),
+                new FieldSelectorDefinition(),
+                // new TypeConverterDefinition(),
```
Comment (hmdebenque) [inline components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/ProcessingFamilyDefinition.java]: Commented code :scream:

Comment (RyanSkraba) [reply components/components-processing/processing-definition/src/main/java/org/talend/components/processing/definition/ProcessingFamilyDefinition.java]: This is temporary -- there was a commit that mixed up the component to be hidden!  It is fixed by https://github.com/Talend/components/pull/1158

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/1777/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/1777/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Arskraba%2Fchore%2Ffix-typo)

**Pull request merged**